templates/digital-asset-template/frontend/pages/Mint/components/BannerSection.tsx

```diff
@@ -0,0 +1,61 @@
+import { Token } from "@/graphql/indexer";
+import { useMemo } from "react";
+import { Image } from "@/components/ui/image";
+
+interface BannerSectionProps {
+  tokens: Token[];
+  className?: string;
+}
+
+export const BannerSection: React.FC<BannerSectionProps> = ({
+  tokens,
+  className,
+}) => {
+  const repeatedTokens = useMemo(() => {
+    let res: Token[] = [];
+    while (res.length < 60) {
+      if (!tokens.length) break;
+      res = res.concat(tokens ?? []);
+    }
+    return res;
+  }, [tokens]);
+
+  return (
+    <div className={className} style={{ position: "relative", left: -50 }}>
+      <div className="flex flex-row gap-4">
+        {repeatedTokens.slice(0, 30).map((token, i) => (
+          <Image
+            rounded
+            src={token.cdn_asset_uris?.cdn_image_uri}
+            style={{
+              aspectRatio: "1/1",
+              width: 136,
+            }}
+            key={
+              i + " " + token.cdn_asset_uris?.cdn_image_uri ??
+              token.token_name ??
+              i
+            }
+          />
+        ))}
+      </div>
+      <div className="flex flex-row gap-4 mt-4">
+        {repeatedTokens.slice(30).map((token, i) => (
+          <Image
+            rounded
+            src={token.cdn_asset_uris?.cdn_image_uri}
+            style={{
+              aspectRatio: "1/1",
+              width: 136,
+            }}
+            key={
+              i + " " + token.cdn_asset_uris?.cdn_image_uri ??
+              token.token_name ??
+              i
+            }
+          />
+        ))}
+      </div>
+    </div>
```

Comment:
I took a light stab at it with grid, realized i forgot the syntax, fumbled around, and then got lazy and used flex. @blakezimmerman, opinions?

Reply:
I pushed a small commit to change this to a grid-based implementation. It simplified the code a little bit.